src/shared/pkcs11-util.c

```diff
@@ -1326,6 +1341,7 @@ int pkcs11_crypt_device_callback(
                         "cryptsetup.pkcs11-pin",
                         data->until,
                         data->headless,
+                        data->askpw_flags,
```

Comment:
The order of the two args is inverted.

Reply:
ouch ouch ouch. A pair of sharp eyes you got there!

Kinda sucks that the compiler didn't mind that we pass a bool as enum type and an enum type as bool here.